### PR TITLE
Add descriptions on module configuration page

### PR DESCRIPTION
**Pull request by jprotin**
Bonjour,

Ajout de plusieurs descriptions sur la configuration du module
Ajout de nouvelle traduction 

Merci beaucoup pour la validation.

Cordialement,

Johan PROTIN

### CODE REVIEW

**Comment by jprotin** [comment]
Hello,

Possible to validate this small upgrade ?

Thank you.

Johan PROTIN

**Comment by Quetzacoalt91** [comment]
Hi @jprotin,

Sorry for the delay. I see that your text is in French, but it should be written in English first and then translated in your favourite language.

In consequence, the text is *hipay.php* must be in English, and then the file *fr.php* should be provided in this PR.

Best regards

**Comment by jprotin** [comment]
Hi @Quetzacoalt91,
It's ok, i create a new tag 1.6.11.

I also applied some corrections.
The next version will be a version with the architecture of Prestashop 1.5 / 1.6 and will arrive mid- February 2016

Thx.

------------------------------------------------------------------------------------------------------------------------------------

Bonjour @Quetzacoalt91,

C'est bon, j'ai créé un nouveau tag 1.6.11.

J'ai aussi appliqué quelques corrections. 
La prochaine version sera une version avec l'architecture de Prestashop 1.5/1.6 et arrivera mi-février 2016

Merci.

File: hipay.php [hunk 1]
```diff
@@ -57,7 +58,7 @@ public function __construct()
 	{
 		$this->name = 'hipay';
 		$this->tab = 'payments_gateways';
-		$this->version = '1.6.9';
+		$this->version = HIPAY_VERSION;
```
Comment (Quetzacoalt91) [inline hipay.php]: With this change, some features could not work properly. The [validator](http://validator.prestashop.com) warns about this point btw.

Bring back the value on this line, on then call it when needed with `$this->version`

File: hipay.php [hunk 2]
```diff
@@ -172,13 +173,14 @@ private function _createAuthorizationOrderState()
...
-				$object->upgrade_detail['1.6.9'][] = 'Erreur sur la mise à jour du statut de commande - En attente de paiement HiPay.';
+				// trasnlate - Erreur sur la mise à jour du statut de commande - En attente de paiement HiPay.
+				$object->upgrade_detail[HIPAY_VERSION][] = $this->l('Error on the order status update - Pending Payment HiPay.');
 				return false;
 			}
 		}
 		if (!Configuration::get('HIPAY_VERSION'))
 		{
-			Configuration::updateValue('HIPAY_VERSION', '1.6.9');
+			Configuration::updateValue('HIPAY_VERSION', HIPAY_VERSION);
```
Comment (Quetzacoalt91) [inline hipay.php]: You can get the module version with 

```php
Configuration::updateValue('HIPAY_VERSION', $this->version);
```

File: hipay.php [hunk 3]
```diff
@@ -172,13 +173,14 @@ private function _createAuthorizationOrderState()
 				copy(dirname(__FILE__).'/logo.gif', dirname(__FILE__).'/../../img/os/'.(int)$os->id.'.gif');
 			}
 			else{
-				$object->upgrade_detail['1.6.9'][] = 'Erreur sur la mise à jour du statut de commande - En attente de paiement HiPay.';
+				// trasnlate - Erreur sur la mise à jour du statut de commande - En attente de paiement HiPay.
+				$object->upgrade_detail[HIPAY_VERSION][] = $this->l('Error on the order status update - Pending Payment HiPay.');
```
Comment (Quetzacoalt91) [inline hipay.php]: You can get the module version with 

```php
Configuration::updateValue('HIPAY_VERSION', $this->version);
```

**Comment by Quetzacoalt91** [comment]
Hi @jprotin,

I see that the config.xml file is not updated in this PR. Can you also change the version inside it please ?

Best regards

**Comment by jprotin** [comment]
Hi @Quetzacoalt91 ,

It's ok : 
- upgrade variables HIPAY_VERSION
- upgrade config.xml

thx

**Comment by jprotin** [comment]
Hi,

Possible to validate this version 1.6.13 ?

Thanks

**Comment by jprotin** [comment]
up ! :)

**Comment by mmoustai** [comment]
Thank you for your contribution.
:+1:

**Pull request merged**